pkg/gpu/ebpf/c/types.h

```diff
@@ -1,4 +1,5 @@
-#pragma once
```

Comment:
:question: Is `#pragma once` not supported? Asking to know what to do for new files.

Reply:
it is supported by the compiler we are using, but i just followed the same pattern as the rest of the .h files in our repo for parity